src/main/resources/templates/fragments/totploginform.html

```diff
@@ -129,13 +129,13 @@ <h3 class="text-center text-with-mdi">
 
                 <script type="text/javascript" th:inline="javascript">
                     /*<![CDATA[*/
-                    var i = /*[[@{#{screen.twofactor.button.loginwip}}]]*/
-                    var j = /*[[@{#{screen.twofactor.button.login}}]]*/
-                        /*]]>*/
-                        $(window).on('pageshow', function () {
-                            $(':submit').prop('disabled', false);
-                            $(':submit').attr('value', j);
-                        });
+                        var i = /*[[@{#{screen.twofactor.button.verifywip}}]]*/
+                        var j = /*[[@{#{screen.twofactor.button.verify}}]]*/
+                    /*]]>*/
```

Comment:
In addition to the indentation issue you mentioned, I should add a default value with `;` at the end for two benefits:

* The default value is displayed in case the the variable is not available
* It makes the template complies with HTML format / grammar